Assets/Scripts/Lever.cs

```diff
@@ -2,24 +2,35 @@
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.Events;
+using UnityEngine.Assertions;
+
+[RequireComponent(typeof(Animator))]
```

Comment:
This is actually an attribute for the class, so I think it's better to remove the line between.

Reply:
I think I wasn't clear on which line. My bad. The `RequireComponent` should go right above the `public class ...` because it's an attribute for class. So, right between `public class ...` and the comment end `*/`.